frontend/src/app/pages/exercises/exercise/shared/simulated-region-overview/tabs/behavior-tab/behaviors/treat-patients/simulated-region-overview-behavior-treat-patients.component.html

```diff
@@ -44,6 +47,19 @@ <h5 class="mt-2">Allgemein</h5>
             ></app-treatment-status-badge>
         </div>
     </div>
+    <div
+        class="row align-items-center"
+        #collapse="ngbCollapse"
+        [(ngbCollapse)]="informationCollapsed"
+        *ngIf="
+            timeUntilNextRecalculation$ | async as timeUntilNextRecalculation
+        "
+    >
+        <div class="col-sm">Nächste Zuteilungs-Berechnung in</div>
+        <div class="col-sm">
+            {{ timeUntilNextRecalculation | formatDuration }}
+        </div>
+    </div>
```

Comment:
Maybe add a small margin to the element above

Reply:
Added in https://github.com/hpi-sam/digital-fuesim-manv/pull/862/commits/5a716dad2a431355b98fe09364f84efd6d94cd4a